### PR TITLE
task/DSAPP-46, task/WP-611: Tools & Apps Workspace Changes; Notifications Enhancements

### PR DESCRIPTION
**Pull request by rstijerina**
## Overview: ##

Various Tools & Apps Workspace changes and notifications enhancements, as requested by Ellen and CMD.

## PR Status: ##

* [X] Ready.

## Related Jira tickets: ##

* [DSAPP-46](https://tacc-main.atlassian.net/browse/DSAPP-46)
* [WP-611](https://tacc-main.atlassian.net/browse/WP-611)

## Summary of Changes: ##
- Support `.log` file previews
- Change "Account Profile" to "Manage Account"
- Add workspace links to account dropdown
- Change `/rw/workspace` path and links to `/workspace`, but support backwards compatibility
- Add "Job Status" label to notification bell
- Sync Job Status notification unread state between react client and angular client, to sync the workspace badge and the header notification bell badge counter
- Adjust Toast UI to look like SecondaryButton

## Testing Steps: ##
1. Confirm UI appears as in photos below for notification bell dropdown, and account dropdown.
2. Confirm both `/rw/workspace` and `/workspace` links work.
3. Confirm Job Status badge counter resets for both the top-right notification bell and the workspace bell whenever either one is clicked.
4. Confirm Toast UI appears as screenshots below after submitting a job

## UI Photos:
- Link UI
Screenshot 2024-09-18 at 8 53 53 PM
Screenshot 2024-09-18 at 8 54 04 PM

- Toast UI
Screenshot 2024-09-18 at 9 46 36 PM
Screenshot 2024-09-18 at 9 46 32 PM

### CODE REVIEW

File: designsafe/apps/signals/websocket_consumers.py [hunk 1]
```diff
@@ -1,28 +1,37 @@
...
         await self.channel_layer.group_discard(
-            self.broadcast_channel, self.channel_name
+            f"ds_{self.scope['user']}", self.channel_name
         )
+        await self.channel_layer.group_discard("ds_broadcast", self.channel_name)
 
-    async def receive(self, text_data):
-        pass
+    async def receive(self, text_data=None, bytes_data=None):
+        if text_data == "markAllNotificationsAsRead":
```
[the remark this answers is not in the document]
Comment (rstijerina) [reply designsafe/apps/signals/websocket_consumers.py:24]: Yes the mark read state is persisted already via the requests, so these websocket messages only need to communicate that to each other.

**Review by chandra-tacc — COMMENTED** [review]
I could not get the notifications for job status working. I can see the hearbeat messages for ngrok. The request for notification (after job submission) did not happen from looking at network log.
Once I visit job status page, I saw the settings for mark all notification read happen, so that part is working.

Rest of the features are working.

**Comment by chandra-tacc** [comment]
> I could not get the notifications for job status working. I can see the hearbeat messages for ngrok. The request for notification (after job submission) did not happen from looking at network log. Once I visit job status page, I saw the settings for mark all notification read happen, so that part is working.
> 
> Rest of the features are working.
> 
> Edit: I'll continue to debug this on my end.

After syncing the webhook url with WEBHOOK_POST_URL in designsafe.env, things are working as normal.

**Review by chandra-tacc — APPROVED** [review]
LGTM. Tested locally.

**Review by wesleyboar — COMMENTED** [review]
✅ I reviewed the UI of toast on hover. **Looks good.** _Better hover state than actual secondary buttons in DS._ Thanks.

**Pull request merged**